requirements.txt

```
@@ -1,6 +1,6 @@
 bs4
 datasets
-git+https://github.com/huggingface/huggingface_hub
+git+https://github.com/huggingface/huggingface_hub.git@fix-auth-in-lfs-upload

```

```suggestion
huggingface_hub>=0.10.0
```

Should be this now, no?